src/main/java/org/vanilladb/core/query/planner/Verifier.java

```diff
@@ -210,10 +221,13 @@ public static void verifyCreateIndexData(CreateIndexData data,
 		Map<String, IndexInfo> indexInfoes = VanillaDb.catalogMgr().getIndexInfo(
 				tableName, tx);
 		if (indexInfoes.containsKey(fieldName))
-			throw new BadSemanticException("field" + fieldName
+			throw new BadSemanticException("field " + fieldName
 					+ " has already been indexed");
 	}
 
+	public static void verifyDropIndexData(DropIndexData data, Transaction tx) {
```

Comment:
What is the function `getIndex` supposed to return? `IndexInfo` or `Index`?
I think it might be easy to implement (?